squirrel.frontier/src/main/java/org/dice_research/squirrel/frontier/impl/FrontierImpl.java

```diff
@@ -29,24 +28,34 @@
  */
 public class FrontierImpl implements Frontier {
 
+    /**
+     * Default value for {@link #generalRecrawlTime} (one week).
+     */
+    public static final long DEFAULT_GENERAL_RECRAWL_TIME = 18000;
```

Comment:
The comment does not fit to the value. 18000ms = 180s = 3min != 1 week. However, one week is more realistic as a default.